mava/utils/environments/debugging_utils.py

```diff
@@ -63,6 +66,9 @@ def make_environment(
             env_module, return_state_info=return_state_info
         )
 
+    if concat_agent_id:
```

Comment:
This might not work for all environments. In some future environments, the features might not be a 1D vector. So we can maybe just keep it at the environment level?

Reply:
Why is it needed that every environment has a `concat_agent_id`?

Reply:
Yes actually the make_environment functions are loose and we expect the user to be more informed about his environment. I only included this here to be general but the default value is False. I thought concatenating agent id and previous actions were tricks we wanted to make available to any environment via an environment wrapper. Then it's up to the user to use this wrapper or not, if the wrapper is actually compatible with his environment. Because I think in general the user has to provide this make_environment function with all the tools to wrap his environment.